src/tests/functional/page_snapshot_tests.ts

```diff
@@ -0,0 +1,53 @@
+import "../helpers/trusted_type_setup"
+import { test } from "@playwright/test"
+import { assert } from "chai"
+
+test.beforeEach(async ({ page }, workerInfo) => {
+  console.log(workerInfo.project.name)
```

Comment:
did you mean to leave in this console message?

Reply:
Whoops, good catch!